README.md

```diff
@@ -1,46 +1,37 @@
-# Electron Installer Grunt Plugin
+# Electron Installer
 
-[![Build status](https://ci.appveyor.com/api/projects/status/yd1ybqg3eq397i26/branch/master?svg=true)](https://ci.appveyor.com/project/kevinsawicki/grunt-electron-installer/branch/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/yd1ybqg3eq397i26/branch/master?svg=true)](https://ci.appveyor.com/project/electronjs/windows-installer/branch/master)
 
-Grunt plugin that builds Windows installers for
+NPM module that builds Windows installers for
 [Electron](https://github.com/atom/electron) apps using
 [Squirrel](https://github.com/Squirrel/Squirrel.Windows).
 
 ## Installing
 
 ```sh
-npm install --save-dev grunt-electron-installer
+npm install --save-dev electron-installer-windows
```

Comment:
I think `electron-windows-installer` might be a slightly clearer name, what do you think?